src/sdk/object_sdk.js

```diff
@@ -801,8 +801,22 @@ class ObjectSDK {
     ////////////
 
     async list_buckets() {
-        const bs = this._get_bucketspace();
-        return bs.list_buckets(this);
+        const start_time = Date.now();
+        let error = 0;
+        try {
+            const bs = this._get_bucketspace();
+            const reply = bs.list_buckets(this);
+            return reply;
+        } catch (e) {
+            error = 1;
+            throw e;
+        } finally {
+            stats_collector.instance(this.internal_rpc_client).update_ops_counters({
+                time: Date.now() - start_time,
+                op_name: `list_bucket`,
```

Comment:
list bucket is not the same as list buckets,
change it to list buckets as the name of this function